contracts/escrow/LemonadeEscrowFactoryV1.sol

```diff
@@ -25,6 +33,22 @@ contract LemonadeEscrowFactoryV1 is Ownable, ILemonadeEscrowFactory {
         return _signer;
     }
 
+    function setFeeCollector(address feeCollector) public onlyOwner {
+        _feeCollector = feeCollector;
+    }
+
+    function getFeeCollector() public view returns (address) {
```

Comment:
Public variables don't need a view function.

Reply:
I would like to keep the `getSigner` as part of the `ILemonadeEscrowFactory` interface, although this signer is also public variable, wdyt?

For other public variables I'll remove the view function